trestlebot/cli/commands/sync_cac_catalog.py

```diff
@@ -0,0 +1,67 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2024 Red Hat, Inc.
+
+"""Module for sync cac catalog command"""
+import logging
+from pathlib import Path
+from typing import Any, List
+
+import click
+
+from trestle.oscal import common
+from trestle.common import const
+from trestlebot.cli.options.common import common_options, git_options, handle_exceptions
+from trestlebot.cli.utils import run_bot
+from trestlebot.tasks.authored.catalog import AuthoredCatalog
+from trestlebot.tasks.base_task import TaskBase
+from trestlebot.tasks.sync_cac_catalog_task import SyncCacCatalogTask
+
+
+logger = logging.getLogger(__name__)
+
+
+@click.command(
+    name="sync-cac-catalog",
+    help="Transform CaC profile to OSCAL catalog.",
+)
+@click.pass_context
+@common_options
+@git_options
+@click.option(
+    "--cac-content-root",
+    help="Root of the CaC content project.",
+    type=click.Path(exists=True, file_okay=False, dir_okay=True),
+    required=True,
+)
+@click.option(
+    "--cac-control",
+    type=str,
+    help="Name of the CaC control file.",
+    required=True,
+)
+@click.option(
```

Comment:
Should this be a required flag? Wondering if you can use the source `policy id` as default name. I only bring it up because it deviates from the other two sync-cac-content implementations. Where the names are set from the cac-content ids.

Reply:
I previously chose not to use policy id because ControlsManager gives up if any policy file can't be parsed, but I agree having similar command usage is more important